src/components/Tokens/TokenTable/TokenTable.tsx

```diff
@@ -92,15 +128,39 @@ export default function TokenTable() {
     } else {
       return (
         <>
-          <GridContainer>
+          <GridContainer fixedHeight={true}>
             <HeaderRow />
-            <TokenRowsContainer>
-              {tokens?.map((token, index) => (
-                <LoadedRow key={token?.name} tokenListIndex={index} tokenListLength={tokens.length} token={token} />
-              ))}
-            </TokenRowsContainer>
+            <TokenDataContainer>
+              <AutoSizer>
+                {({ height, width }) => (
+                  <InfiniteLoader
+                    isItemLoaded={isItemLoaded}
+                    itemCount={MAX_TOKENS_TO_LOAD}
+                    loadMoreItems={loadMoreTokens}
+                  >
+                    {({
+                      onItemsRendered,
+                      ref,
+                    }: {
+                      onItemsRendered: (props: ListOnItemsRenderedProps) => any
+                      ref: any
+                    }) => (
+                      <FixedSizeList
+                        height={height}
+                        width={width}
+                        itemCount={MAX_TOKENS_TO_LOAD}
+                        itemSize={70}
+                        onItemsRendered={onItemsRendered}
+                        ref={ref}
+                      >
+                        {Row}
```

Comment:
not sure about the `FixedSizeList` api, but I think you should be able to use the function above as a normal component here. <Row />

Reply:
this is going to get changed in fix for this: https://uniswaplabs.atlassian.net/browse/WEB-1149?atlOrigin=eyJpIjoiMWE1NjEzM2Y5M2NjNDBmZmJkMmE0YjQzMTE1ZTFlZjUiLCJwIjoiaiJ9 so will leave this for future PR